go.mod

```
@@ -1,15 +1,23 @@
 module github.com/stackrox/ossls
 
-go 1.14
+go 1.21.9
```

```suggestion
go 1.21

toolchain go1.21.9
```

This way users of this (if this were used as a library) are not required to use go1.21.9+, but rather any go1.21.0+. Setting `toolchain` means that go version is required when this is run as the main module.